packages/config/src/config/tile.set.ts

```diff
@@ -26,6 +26,9 @@ export interface ConfigLayer extends Partial<Record<EpsgCode, string>> {
 
   /** Max zoom to show the layer @default 30 */
   maxZoom?: number;
+
+  /** Disable the layer from the TileSet @default false*/
+  disable?: boolean;
```

Comment:
I think we should use `disabled` not `disable` 

`disabled` is what all the HTML elements use and it has a slightly different dictionary meaning